chapters/02_hello.md

```diff
@@ -98,7 +36,7 @@ Melonhead began tracking the patterns we do at  Thursday and Sunday MAJ practice
 an amazing collection, and for those who remember the code words, like  *1-5-1 Foie Graham* or *4-5 Twister*, it's enough
 to figure out what we really did.  But most of us forget the code words after a while,
 except for the frequently juggled ones, and this was impetus for the  Version 3.0
-update by Mike Newton in 2018.  Since then, we've suffered through two years of COVID restrictions and two lost Madfests.   But we kept juggling and discovered some cool new patterns that work well with face masks!  Nathan Nard's rejuvenating and inventive contributions have helped to launch this recent Version 3.1.
+update by Mike Newton in 2018.  Since then, we've suffered through two years of COVID restrictions and two lost Madfests.   But we kept juggling and discovered some cool new patterns that work well with face masks!  Nathan Nard's rejuvenating and inventive contributions have helped to launch this recent Version \majpatternbookversion.
```

Comment:
purpose of \majpatternbookversion to help reduce "hunting down" outdated version numbers to update them.  It's a little finicky though, may not be worth it.  Totally fine with removing this if you find it unnecessary.